Essentials/src/main/resources/messages.properties

```diff
@@ -284,19 +411,27 @@ helpOp=\u00a74[HelpOp]\u00a7r \u00a76{0}\:\u00a7r {1}
 helpPlugin=\u00a74{0}\u00a7r\: Plugin Help\: /help {1}
 helpopCommandDescription=Message online admins.
 helpopCommandUsage=/<command> <message>
+helpopCommandUsage1=/<command> <message>
+helpopCommandUsage1Description=Sends the given message to all online admins
 holdBook=\u00a74You are not holding a writable book.
 holdFirework=\u00a74You must be holding a firework to add effects.
 holdPotion=\u00a74You must be holding a potion to apply effects to it.
 holeInFloor=\u00a74Hole in floor\!
 homeCommandDescription=Teleport to your home.
 homeCommandUsage=/<command> [player:][name]
+homeCommandUsage1=/<command> <name>
+homeCommandUsage1Description=Teleports you to your home with the given name
+homeCommandUsage2=/<command> <player>:<name>
+homeCommandUsage2Description=Teleports you to the specified player's home with the given name
 homes=\u00a76Homes\:\u00a7r {0}
 homeConfirmation=\u00a76You already have a home named \u00a7c{0}\u00a76!\nTo overwrite your existing home, type the command again.
 homeSet=\u00a76Home set to current location.
 hour=hour
 hours=hours
 ignoreCommandDescription=Ignore or unignore other players.
 ignoreCommandUsage=/<command> <player>
+ignoreCommandUsage1=/<command> <player>
+ignoreCommandUsage1Description=Ignores/Unignores the given player
```

Comment:
I think this should be "Ignores/unignores" or "Ignores or unignores" (and likewise for all similar messages).

Reply:
```suggestion
ignoreCommandUsage1Description=Ignores or unignores the given player
```